flink-ml-lib/src/main/java/org/apache/flink/ml/feature/randomsplitter/RandomSplitter.java

```diff
@@ -83,11 +84,12 @@ public Table[] transform(Table... inputs) {
 
     private static class SplitterOperator extends AbstractStreamOperator<Row>
             implements OneInputStreamOperator<Row, Row> {
-        private final Random random = new Random(0);
+        private final Random random;
         OutputTag<Row>[] outputTag;
         final double[] fractions;
 
-        public SplitterOperator(OutputTag<Row>[] outputTag, Double[] weights) {
+        public SplitterOperator(OutputTag<Row>[] outputTag, Double[] weights, long seed) {
+            random = new Random(seed);
```

Comment:
It might be better to avoid having random values behaving the same on each subtask, e.g., always assigning the first element to the first output table regardless of the id of the subtask. You may check [`RowGenerator.open()`](https://github.com/apache/flink-ml/blob/master/flink-ml-benchmark/src/main/java/org/apache/flink/ml/benchmark/datagenerator/common/RowGenerator.java#L52) for how to create randoms from both initial seed and subtask id.

Reply:
Sounds good!